src/Tasks/Microsoft.NET.Build.Tasks/IPackageResolver.cs

```diff
@@ -8,5 +8,6 @@ namespace Microsoft.NET.Build.Tasks
     public interface IPackageResolver
     {
         string GetPackageDirectory(string packageId, NuGetVersion version);
+        string GetPackageDirectory(string packageId, NuGetVersion version, out string packageRoot);
```

Comment:
This one is only breaking if someone *implements* the interface, which I doubt anyone would do, but my experience with these sorts of things on corefx was traumatic.